Rocket.Chat/Controllers/Auth/WebViewController.swift

```diff
@@ -19,6 +19,12 @@ class WebViewController: OAuthWebViewController {
 
     var activityIndicator: UIActivityIndicatorView!
 
+    convenience init(authorizeURL: URL?) {
+        self.init()
+        self.targetURL = authorizeURL
+        self.viewDidLoad()
```

Comment:
Should we be calling it in init?

Reply:
That's a workaround to ensure the WKWebView is loaded, unfortunately :)

I'll see what I can do.